08LessonEight/homework/css/exercise-06-style.css

```diff
@@ -1 +1,36 @@
-/*Add CSS rules*/
\ No newline at end of file
+* {box-sizing: border-box;}
+body {
+  font-family: helvetica;
+  color: #555555;
+  max-width: 800px;
+  margin: 0 auto;
+}
+.parent {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 20px;
+}
+section {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+article {
+  margin-right: 25px;
+}
+img {
+height: 200px;
+}
+button {
+  border-left: solid 1px;
+  border-top: solid 1px;
+  border-bottom: solid 1px;
+  background: transparent;
+  font-weight: bold;
+  color: gray;
+  border-right: solid #8E1E02 4px;
```

Comment:
Try to keep your similar elements together; i.e. move your `border-right` up with the other border declarations.